ansible/roles/pusher/tasks/main.yml

```diff
@@ -9,17 +9,3 @@
 - name: Link docker push central registry secret
   command: "{{ oc }} secrets link builder push-external-registry -n {{ project_name }}"
   when: central_registry_secret|failed
-- name: Check if push prod registry secret exists
-  command: "{{ oc }} get secret push-prod-registry -n {{ project_name }}"
-  register: prod_registry_secret
-  ignore_errors: true
-- name: Docker push prod registry secret
-  command: "{{ oc }} secrets new-dockercfg push-prod-registry --docker-username=unused --docker-password={{ hostvars['prod']['pushtoken'] }} --docker-email=unused -n {{ project_name }}"
```

Comment:
@codificat there is a buildconfig that outputs to the external registry wouldn't this still be needed?

Reply:
@jcpowermac AFAICT output goes to the `central_registry` but `prod_registry` is not actually used/referenced... so the suggestion here is to remove it.... did I miss something?

Reply:
@codificat it would have helped if I read all the tasks.  Were good.